Source/WebCore/accessibility/isolatedtree/mac/AXIsolatedObjectMac.mm

```diff
@@ -343,24 +343,24 @@
     if (!attributedStringContainsRange(attributedText.get(), *nsRange))
         return nil;
 
-    NSMutableAttributedString *result = [[NSMutableAttributedString alloc] initWithAttributedString:[attributedText attributedSubstringFromRange:*nsRange]];
-    if (!result.length)
+    RetainPtr result = adoptNS([[NSMutableAttributedString alloc] initWithAttributedString:[attributedText attributedSubstringFromRange:*nsRange]]);
+    if (!result.get().length)
         return result;
 
-    auto resultRange = NSMakeRange(0, result.length);
+    auto resultRange = NSMakeRange(0, result.get().length);
     // The AttributedString is cached with spelling info. If the caller does not request spelling info, we have to remove it before returning.
     if (spellCheck == SpellCheck::No) {
-        [result removeAttribute:NSAccessibilityDidSpellCheckAttribute range:resultRange];
-        [result removeAttribute:NSAccessibilityMisspelledTextAttribute range:resultRange];
-        [result removeAttribute:NSAccessibilityMarkedMisspelledTextAttribute range:resultRange];
+        [result.get() removeAttribute:NSAccessibilityDidSpellCheckAttribute range:resultRange];
+        [result.get() removeAttribute:NSAccessibilityMisspelledTextAttribute range:resultRange];
+        [result.get() removeAttribute:NSAccessibilityMarkedMisspelledTextAttribute range:resultRange];
```

Comment:
`.get()` is unnecessary when making method calls.

Reply:
oh, good point.